Examples/create-windows-cloud-image.ps1

```diff
@@ -39,12 +42,24 @@ $extraDriversPath = "C:\drivers\"
 # Usually, the second image version is the Standard one
 $image = (Get-WimFileImagesInfo -WimFilePath $wimFilePath)[1]
 
+
+# The path were you want to create the config fille
+$ConfigFilePath =".\config.ini"
+
+New-WindowsImageConfig -ConfigFilePath $ConfigFilePath
+
+$fCfgPath = Resolve-Path $ConfigFilePath
+
```

Comment:
Add a comment,saying:
This is an example how to automate the image configuration file according to your needs